content/modules/webdav/pages/configuration.adoc

```diff
@@ -177,6 +177,32 @@ server.ssl.key-alias = localhost
 server.port = 8443
 ----
 
+=== Install trusted certificate
```

Comment:
Move this section before `Configuring Server` - so there will be 2 ways of keystore creation and configuration as a last step.